dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentFactoryImpl.java

```diff
@@ -2052,6 +2042,25 @@ private static String findAndReplaceQueryDates(String query) {
                         // Format MM/dd/yyyy
                         replace = replaceDateWithFormat(replace, "\\\"?(\\d{1,2}/\\d{1,2}/\\d{4})\\\"?");
```

Comment:
shouldn't have unit test this?

Reply:
it could be a Map<String, String> replaceTimeFormatMap = imap ("\\\"?(\\d{1,2}:\\d{1,2}:\\d{1,2}(?:AM|PM|am|pm))\\\"?", "hh:mm:ssa", ....);

Reply:
then, for (final Map.Entry<String, String> entry : replaceTimeFormatMap.entrySet()) {

replace = replaceTimeWithFormat(replace, entry.getKey(), entry.getValue());
}